pkg/translator/prometheusremotewrite/helper.go

```diff
@@ -204,6 +213,14 @@ func createAttributes(resource pcommon.Resource, attributes pcommon.Map, externa
 		}
 	}
 
+	// add scope name/version labels
+	if len(scope.Name()) > 0 {
+		extras = append(extras, scopeAttrName, scope.Name())
+	}
+	if len(scope.Version()) > 0 {
```

Comment:
Should version be added as a label if name does not exist?  That implies bad data to me and I would omit both values.

Reply:
I looked and most exporters in contrib will add the two independently, however I did find [one place](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/translator/loki/encode.go#L115) where `version` is excluded if `name` isn't present.

That said, I'm not clear on what the right solution is here. [Instrumentation Scope](https://opentelemetry.io/docs/specs/otel/glossary/#instrumentation-scope) is defined as identified through `name` or a `(name,version)` pair, so `version` on its own doesn't seem like it would be helpful. However, the guide for [mapping OTLP to non-OTLP](https://opentelemetry.io/docs/specs/otel/common/mapping-to-non-otlp/#instrumentationscope) doesn't mention that `name` is required for `version` to be present, and while it doesn't directly apply here, the [proto definitions say](https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/common/v1/common.proto#L72) that an empty `name` field means it is unknown.

I'd probably lean toward saying we should just include the version, though I'm pretty conflicted.

Reply:
I'm also in-favor of including version if the name is not present.  Unless it is explicitly prohibited by the spec, we should be flexible with what we accept here.

Reply:
The outcome from the 2023-08-02 NA/EU Collector SIG meeting discussion on this is that we would like to emit this data regardless of whether it is correct.